core/src/main/java/feast/core/service/ProjectService.java

```diff
@@ -16,35 +16,58 @@
  */
 package feast.core.service;
 
+import feast.core.auth.authorization.AuthorizationProvider;
+import feast.core.auth.authorization.AuthorizationResult;
+import feast.core.config.FeastProperties;
+import feast.core.config.FeastProperties.SecurityProperties;
 import feast.core.dao.ProjectRepository;
 import feast.core.model.Project;
 import java.util.List;
 import java.util.Optional;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.ObjectProvider;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
 import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
 
 @Slf4j
 @Service
-public class AccessManagementService {
+public class ProjectService {
```

Comment:
Not sure about the change of naming here. Here we are mostly dealing with projects in the context of authentication so it makes more sense to be `AccessManagementService` or `AuthorizationService` instead of `ProjectService`.

Reply:
@woop was this change done by you? if not it must be merge/rebase problem.

Reply:
This was done from my side. I am still torn between these two. `ProjectService` is more explicit, but the idea was always that it would be generalized later. I just felt like `AccessManagementService` is a premature generalization.